lib/iex/lib/iex/introspection.ex

```diff
@@ -644,7 +644,15 @@ defmodule IEx.Introspection do
         types_not_found(inspect(module))
 
       {:ok, types} ->
-        Enum.each(types, &(&1 |> format_type() |> IO.puts()))
+        types
+        |> Enum.sort_by(fn {_, {name, _, args}} -> {name, length(args)} end)
+        |> case do
+          [] ->
```

Comment:
I think this clause can never be executed :)

Reply:
Without the `typep` filter, you're correct. I'll remove the case.

Reply:
Fixed in 9830d3d8089f6594a821a5fe39ccf2be0247d725